picard/ui/logview.py

```diff
@@ -37,7 +41,11 @@ def __init__(self, title, logger, w=740, h=340, parent=None):
         self.browser.setDocument(self.doc)
         self.vbox = QtWidgets.QVBoxLayout(self)
         self.vbox.addWidget(self.browser)
-        self._display()
+        self._setup_formats()
+        self.verbosity = self.default_verbosity
```

Comment:
Dont really get the use of the debug checkbox and this drop down. We can keep only this and remove the checkbox?

Reply:
> Dont really get the use of the debug checkbox and this drop down. We can keep only this and remove the checkbox?

That's 2 different things, once again, you mix up the logger and the log viewer:
- debug mode controls if we log debug messages (if disabled, they aren't written anywhere at all)
- verbosity debug unchecked will hide the debug messages from the view, but they are still outputted to console for example

Reply:
Hmm, can we separate the logger config from the log viewer config?

Reply:
> Hmm, can we separate the logger config from the log viewer config?

What do you mean ?

Debug mode is just a trick to allow user to enable the debug mode from within the app (before it was only possible using the -d command line option).
Logging level setting could be moved elsewhere in Options, but then it would be a bit cumbersome to toggle debug mode on/off.

Think about the end user:
1. open Log View
2. check Debug mode checkbox
3. do some action
4. press Save as
5. uncheck Debug mode checkbox

Instead of:
1. Open Options
2. go to panel "Advanced"
3. set the log level to Debug (if it wasn't already...)
4. Press OK
5. open the Log View
6. do some action
7. press Save as
8. Open Options
9. go to panel "Advanced"
10. set the log level back to what it was (if you remember)
11. Press OK